scripts/count_opcodes.py

```diff
@@ -237,6 +243,24 @@ def reporting_guts(self, counter, co, bias):
             key = f"!{const!r}"
             counter[key] += 1
 
+class RegistersReporter(Reporter):
+
+    def reporting_guts(self, counter, co, bias):
+        regs = (co.co_argcount +
+                len(co.co_cellvars) +
+                len(co.co_consts) +
+                len(co.co_freevars) +
+                len(co.co_names) +
```

Comment:
I don’t see why names should go in registers. The name itself is not data to be loaded, it is a parameter to the instruction ( e.g. LOAD_ATTR).

same for co_varnames.

And isn’t co_lnlocals the sum of three others?

Reply:
We don't need to merge this PR, it was just to get an idea.  There were very few functions with >255 registers even with all this over-counting, so it's not going to be an issue if we need the occasional EXTENDED_ARGS.